src/app.css

```diff
@@ -20,6 +20,9 @@
     --primary-200: #ff9d00;
     --primary-300: #ffbf00;
     --primary-400: #e3ff00;
+    
+    --primary-500: #3f51b5;
```

Comment:
Instead of defining new css-variables you can use arbitary values with tailwind like `bg-[#3f51b5]`, see their docs [here](https://tailwindcss.com/docs/background-color#arbitrary-values)